templates/docs/patterns/grid/index.md

```diff
@@ -51,6 +51,32 @@ Read also: [Breakpoints](/docs/settings/breakpoint-settings)
     View example of the default grid
 </a></div>
 
+There are some special classes to help you build [common layout patterns](/docs/layouts/brochure). Instead of deciding how many columns each element should use for different screen sizes, use one of these classes on the grid container, and the child elements will be arranged automatically as long as they have the `col` class.
```

Comment:
It seems that this whole page could use some content update now that we introduce these new patterns, but let's not worry about it now.

I would at least give this section a heading like `## Common layouts` (and then change the subheadings to ###)

Reply:
Done! I didn't do it before because if it's an h3 you can't link directly to it, which was Ant's suggestion. It's not a big deal because you can link to `common-patterns` anyway